app/src/constants.js

```diff
@@ -122,6 +122,10 @@ export const REPORT_STATUS = {
   sent: 'sent'
 };
 
+export const VESSEL_INFO_HIDDEN = 1;
+export const VESSEL_INFO_LOADING = 2;
+export const VESSEL_INFO_LOADED = 3;
```

Comment:
In that kind of things I tend to prefer an "enumerable" style, but probably a matter of taste:
```
export const VESSEL_INFO_STATUS = {
  HIDDEN: 1
  ...
} 
...
if (status === VESSEL_INFO_STATUS.HIDDEN) 
...

```